test/test_suit/iceberg_json_aws.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+
+from test_suit.test_utils import RetryableError, NonRetryableError
+import json
+from time import sleep
+from test_suit.base_e2e import BaseE2eTest
+
+
+class TestIcebergJsonAws(BaseE2eTest):
+    def __init__(self, driver, nameSalt: str):
+        self.driver = driver
+        self.fileName = "iceberg_json_aws"
+        self.topic = self.fileName + nameSalt
+
+    def getConfigFileName(self):
+        return self.fileName + ".json"
+
+    def setup(self):
+        self.driver.create_iceberg_table_with_content(
+            table_name=self.topic,
+            external_volume="kafka_push_e2e_volume_aws",  # volume created manually
+        )
+
+    def send(self):
+        msg = json.dumps(
+            {
+                "id": 1,
+                "body_temperature": 36.6,
+                "name": "Steve",
+                "approved_coffee_types": ["Espresso"],
```

Comment:
You are missing a few
```suggestion
                "approved_coffee_types": ["Espresso", "Doppio", "Ristretto", "Lungo"],
```